packages/shared/lib/core/wallet/actions/refreshNativeTokens.ts

```diff
@@ -0,0 +1,16 @@
+import { selectedAccount } from '@core/account'
+import { get } from 'svelte/store'
+import { addNativeTokenAsset, clearNativeTokenAssets } from '../stores'
+
+export function refreshNativeTokens(): void {
+    clearNativeTokenAssets()
+    get(selectedAccount)?.balances?.nativeTokens?.forEach((nativeToken) => {
+        addNativeTokenAsset({
+            id: nativeToken.id,
```

Comment:
```suggestion
            id: nativeToken.tokenId,
```

Wallet.rs class needs to be updated